src/record/record.js

```diff
@@ -449,7 +446,16 @@ Record.prototype._applyUpdate = function( message ) {
  */
 Record.prototype._onRead = function( message ) {
 	this.version = parseInt( message.data[ 1 ], 10 );
-	this._applyChange( jsonPath.set( this._$data, undefined, JSON.parse( message.data[ 2 ] ) ) );
+
+	this._$data = JSON.parse( message.data[ 2 ] );
```

Comment:
Okay, so cases ontop of my head:

###### Old behavior - record is/isn't ready - triggerNow true:
1) Subscribe to path `name`
2) Get read -> {}
3) name triggered with undefined

###### Old behavior - record is/isn't ready - triggerNow false:
1) Subscribe to path `name`
2) Get read -> {}
3) nothing...

###### New behavior - record is ready - triggerNow false:
1) Subscribe to path `name`
2) Nothing..

###### New behavior - record isnt ready - triggerNow false:
1) Subscribe to path `name`
2) Get read -> {}
3) name triggered with undefined

###### New behavior - record is ready - triggerNow true:
1) Subscribe to path `name`
3) name triggered with undefined

###### New behavior - record isnt ready - triggerNow true:
1) Subscribe to path `name`
2) name triggered with undefined
3) Get read -> {}
4) name triggered with undefined